components/OurGoalsAndMissionCardContent/OurGoalsCardContent.js

```diff
@@ -0,0 +1,67 @@
+import { Row, Col } from "../bootstrap"
+import Image from "react-bootstrap/Image"
+import styles from "./OurGoalsAndMissionCardContent.module.css"
+
+const OurGoalsCardContent = (
```

Comment:
Let's make this a function component, change `body` to `children` in `AboutPagesCard`, and render it as a child:

```js
<AboutPagesCard title="Our Goals">
  <GoalsAndMissionCardContent/>
</AboutPagesCard>
```

Reply:
Oh this is nice!  I didn't know about this capability earlier.  Yes much better.